src/controllers/dashboardgeneral.js

```diff
@@ -1,127 +1,51 @@
-define(["jQuery", "loading", "fnchecked", "emby-checkbox", "emby-textarea", "emby-input", "emby-select", "emby-linkbutton"], function($, loading) {
+define(["jQuery", "loading", "fnchecked", "emby-checkbox", "emby-textarea", "emby-input", "emby-select", "emby-button"], function($, loading) {
     "use strict";
 
     function loadPage(page, config, languageOptions, systemInfo) {
-        if (systemInfo.CanLaunchWebBrowser) {
-            page.querySelector("#fldRunWebAppAtStartup").classList.remove("hide");
-        } else {
-            page.querySelector("#fldRunWebAppAtStartup").classList.add("hide");
-        }
-        page.querySelector("#txtCachePath").value = config.CachePath || "";
-        $("#txtMetadataPath", page).val(config.MetadataPath || "");
-        $("#txtMetadataNetworkPath", page).val(config.MetadataNetworkPath || "");
-        $("#selectLocalizationLanguage", page).html(languageOptions.map(function(language) {
-            return '<option value="' + language.Value + '">' + language.Name + "</option>"
-        })).val(config.UICulture);
-        currentLanguage = config.UICulture;
-        if (systemInfo.CanSelfUpdate) {
-            page.querySelector(".fldAutomaticUpdates").classList.remove("hide");
-        } else {
-            page.querySelector(".fldAutomaticUpdates").classList.add("hide");
-        }
-        $("#chkEnableAutomaticServerUpdates", page).checked(config.EnableAutoUpdate);
-        $("#chkEnableAutomaticRestart", page).checked(config.EnableAutomaticRestart);
-        if (systemInfo.CanSelfRestart) {
-            page.querySelector("#fldEnableAutomaticRestart").classList.remove("hide");
-        } else {
-            page.querySelector("#fldEnableAutomaticRestart").classList.add("hide");
-        }
-        if (systemInfo.CanSelfRestart || systemInfo.CanSelfUpdate) {
-            $(".autoUpdatesContainer", page).removeClass("hide");
-        } else {
-            $(".autoUpdatesContainer", page).addClass("hide");
-        }
-        loading.hide();
+        systemInfo.CanLaunchWebBrowser ? page.querySelector("#fldRunWebAppAtStartup").classList.remove("hide") : page.querySelector("#fldRunWebAppAtStartup").classList.add("hide"), page.querySelector("#txtCachePath").value = config.CachePath || "", $("#selectLocalizationLanguage", page).html(languageOptions.map(function(l) {
+            return '<option value="' + l.Value + '">' + l.Name + "</option>"
```

Comment:
This file has been reverted to a previous state.

Reply:
I restored to master version, and then i proceeded in replacing emby-linkbutton to emby-button.